app/resources/forms.js

```diff
@@ -329,3 +329,19 @@ function validate_fields() {
 
   return true;
 }
+
+// Expand the form to see the detail form
+function expand_forms(expandSection) {
+  jQuery(document).ready(function() {
```

Comment:
Optional: I prefer to put this line directly in create.html.template and add_note.html.template instead of putting here. This is like a main() function in C, so it would be easier to read if it's directly defined in each HTML file rather than hiding it inside a function.